dataprofiler/tests/data_readers/test_parquet_data.py

```diff
@@ -130,6 +150,24 @@ def test_data_formats(self):
                     self.assertIsInstance(data, list)
                     self.assertIsInstance(data[0], str)
 
+    def test_data_formats_when_sampled(self):
+        """
+        Determine if the parquet file data_formats can be used when sampled
+        """
+        for input_file in self.file_or_buf_list:
+            input_data_obj = Data(input_file["path"], options={"sample_nrows": 100})
+            for data_format in list(input_data_obj._data_formats.keys()):
+                input_data_obj.data_format = data_format
+                self.assertEqual(input_data_obj.data_format, data_format)
+                data = input_data_obj.data
+                if data_format == "dataframe":
+                    import pandas as pd
```

Comment:
can we not put the `import...` at the top of the file?

Reply:
fixed